<launch>
  <node pkg="kinematics_cache" type="kinematics_cache_node" name="kinematics_cache_node">
    <param name="max_distance" value="0.821000"/>
    <param name="left_arm_data" value="/home/jlurz/octree/left_arm.ot" />
    <param name="right_arm_data" value="/home/jlurz/octree/right_arm.ot" />
  </node>
</launch>
